pkg/sdk/go/errors.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package sdk
+
+import (
+	"github.com/mainflux/mainflux/pkg/errors"
+)
+
+var (
```

Comment:
Some of these errors are not used.

Reply:
I have left just 4, which are being used. I can remove `ErrCerts`.
And for the others, I can move them to errors/types.go , in the errors package.